src/App.tsx

```diff
@@ -1,26 +1,153 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
```

Comment:
I think we could move this whole app.tsx into it's own component,
I say this because later we will want to have routing etc, and an ideal App.tsx would just be:

```tsx
function App() {
  return (<div>
      <Navbar/>
      <Hero/>
      <Router> ... </Router>
  </div>)
}
```